### ISSUE TITLE
Use RTD theme for all documentation compilation

### ISSUE THREAD

**Issue by benjaoming** [learningequality/ka-lite#3598]
Just `pip install sphinx_rtd_theme` and add this:

http://docs.readthedocs.org/en/latest/theme.html

```python
# on_rtd is whether we are on readthedocs.org
import os
on_rtd = os.environ.get('READTHEDOCS', None) == 'True'

if not on_rtd:  # only import and set the theme if we're building docs locally
    import sphinx_rtd_theme
    html_theme = 'sphinx_rtd_theme'
    html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]

# otherwise, readthedocs.org uses their theme by default, so no need to specify it
```

Furthermore, we need to move all wiki pages and update learningequality.org/ka-lite to point people directly to the documentation.

I think the change of theme and moving of wiki pages can safely be backported to the 0.13 branch to ensure that we have all that in all versions of the docs.

Also,

**Comment by MCGallaspy** [comment]
Why use the RTD theme? We're not building on RTD, since it chokes on the screenshot directive (doesn't have the right binaries in its build server, like firefox). As far as I know, ` learningequality.org/ka-lite` does point to the new docs. Also, you have apparently left off the last part of your message.

**Comment by benjaoming** [comment]
No, learningequality.org points to the Github wiki!

**Comment by benjaoming** [comment]
Well, okay it doesn't to that entirely, but some places, for instance on the "Community" part of the front page.

**Comment by benjaoming** [comment]
As for the RTD theme, it's much nicer :) It's also responsive!

What was the problem with screen shots stuff? I'm pretty sure you can include and run your own Python code on RTD?

**Comment by MCGallaspy** [comment]
@benjaoming if I recall correctly, it wasn't the python code, but rather that the build environment simply doesn't have a firefox executable.

**Comment by benjaoming** [comment]
@MCGallaspy ah okay, I was thinking that the docs builder would call a "magic" URL that generated the screenshot on a learningequality.org server? :)

**Comment by benjaoming** [comment]
Fixed in #3966

**Comment by aronasorman** [comment]
Seems like it's resolved in #3966? I don't remember the style changing, but that's just me :)

**Comment by benjaoming** [comment]
Yes, it's been changed so now people won't think our software is from 2008 :P
